Remove SIMPLE from HTML (#38564)
mdn/content: files/en-us/web/html/element/video/index.md

</video>
```

> The above example shows usage of the `<video>` element. Similarly to the {{htmlelement("img")}} element, we include a path to the media we want to display inside the `src` attribute; we can include other attributes to specify information such as video width and height, whether we want it to autoplay and loop, or to show the browser's default video controls, etc.

Reviewer: ```suggestion
The above example shows how to use the `<video>` element. Similar to the {{htmlelement("img")}} element, we include a path to the media we want to display inside the `src` attribute; we can include other attributes to specify information such as video width and height, whether we want it to autoplay and loop, or to show the browser's default video controls, and so on.
```
